src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/ExceptionRecordFinalizer.java

```diff
@@ -13,6 +13,7 @@
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.OCR_DATA_VALIDATION_WARNINGS;
 
 @Service
+// TODO rename to ResponseFinalizer
```

Comment:
I think that would be a bit too generic. Some of our responses are for service cases, not exception records.

Reply:
Ok, will keep the name of the class